plugins/rogue/scripts/heartbeat.sh

```
@@ -32,33 +32,18 @@ set +u
-# Family is the fixed enum value "claude". The surface rides the separate
-# x-rogue-agent header (free-form display label), derived from
-# CLAUDE_CODE_ENTRYPOINT (the same var hook.sh uses to tell GUI from cli).
-# Unknown → CLI.
-case "$(printf '%s' "${CLAUDE_CODE_ENTRYPOINT:-}" | tr '[:upper:]' '[:lower:]')" in
-  *cowork*)  AGENT="Claude Cowork" ;;
-  *desktop*) AGENT="Claude Code - Desktop" ;;
-  *)         AGENT="Claude Code - CLI" ;;
-esac
+# Host + version + surface label. Shared with hook.sh, which sends the same three
+# as headers on every event: same values, same roster row. See install-id.sh.
+[ -r "${CLAUDE_PLUGIN_ROOT:-}/scripts/install-id.sh" ] && . "${CLAUDE_PLUGIN_ROOT}/scripts/install-id.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./scripts/install-id.sh was not specified as input (see shellcheck -x).

(SC1091)